pytorch/inference/docker/1.13/py3/Dockerfile.ec2.cpu.core_packages.json

```diff
@@ -0,0 +1,48 @@
+{
+   "captum":{
+      "version_specifier":"==0.6.0",
+      "skip":"True"
+   },
+   "torchaudio":{
+      "version_specifier":"==0.13.1+cpu",
```

Comment:
Pinning these packages up to the patch version means that future patch version updates to the same framework version would require a change in the "code_packages" file as well.

Is there a test that ensures that these "core_packages" defined here match correctly with the versions installed on the image?

What is the user-experience when there is a mismatch? Does it surface as a failed test with a simple explanation, or does it result in a deep-failure that might surface as a failure in the overall security check workflow that requires close attention from an engineer to trace back to this file?

Reply:
Yes, there is a test that has been added, which checks if the installed package versions and the core package versions match. In case of a mismatch a failed test with a simple explanation would show up.